src/ui/simulator/application/study.cpp

```diff
@@ -19,39 +19,36 @@
 ** along with Antares_Simulator. If not, see <https://opensource.org/license/mpl-2-0/>.
 */
 #include "antares/study/study.h"
+
```

Comment:
The only change here is definition and call to **containsOnlyASCIIcharsOnWindows(...)**.
The rest is automatic reformatting.